_draft-tests/SC2-4-2-title.md

```diff
@@ -0,0 +1,50 @@
+---
+rule_id: SC2-4-2-title
+test_case_template: default
+success_criterion:
+- 2.4.2
+---
+
+### Passed
+
+```html
+<html>
+  <title>This page has a title</title>
+</html>
+```
+
+```html
+<html>
+  <title>This page gives a title to an iframe</title>
+  <iframe src="page-without-title.html"></iframe>
+</html>
+```
```

Comment:
Possibly also worth adding an example with title in a head element and a second title in a body element, as something that would pass, despite less valid HTML. For consistency in test tools.